integration/build_test.go

```diff
@@ -207,6 +205,7 @@ func TestExpectedBuildFailures(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.description, func(t *testing.T) {
+			MarkIntegrationTest(t, NeedsGcp)
```

Comment:
what effect does this change have?

Reply:
> what effect does this change have?


This change uses the hash value of an individual test case name to assign partition instead of the whole test, that is 

``TestExpectedBuildFailures/jib_is_too_old`` vs  ``TestExpectedBuildFailures``, we can have many test cases per test, then we will have better granularity,  for example, a test suite with 10 test case, partitioning by test suite name, will assign all the 10 test cases in this test suite into one partition, partitioning by test case name(this change) will distribute those 10 test case among all partitions we have, the workload is more likely to be balanced

Reply:
Though we currently only have one test case in this test, that doesn't make much difference for partitioning.. but we may have more test cases in the future..

Reply:
got it. That makes perfect sense.